pay-api/src/pay_api/services/payment_service.py

```diff
@@ -64,9 +64,14 @@ def create_payment(cls, payment_request: Tuple[Dict[str, Any]], authorization: T
         account_info = payment_request.get('accountInfo', None)
         filing_id = filing_info.get('filingIdentifier', None)
         folio_number = filing_info.get('folioNumber', get_str_by_path(authorization, 'business/folioNumber'))
+        if account_info is not None and account_info.get('bcolAccountNumber', None) is not None:
```

Comment:
does if account_info  and account_info.get('bcolAccountNumber', None) work? just checking , I am not sure..

Reply:
Yes, that will work too.. We have this thing mixed in many places, we will need to stick with one.